### PR TITLE
Fix TD-4875:Two Scripture guidewords on odd pages

### PR DESCRIPTION
**Pull request by vkarthim**
*Set common frame height like dictionary
*Added condition to skip on special case
*Modified the Nunit test case and fails based on new change

### CODE REVIEW

File: src/OpenOfficeConvert/OOStyles.cs [hunk 1]
```diff
@@ -2123,14 +2123,7 @@ private void CreateRightGuideword(int i)
...
                 _writer.WriteStartElement("draw:text-box");
-                if (_projInfo.ProjectInputType.ToLower() == "dictionary")
-                {
-                    _writer.WriteAttributeString("fo:min-height", "19.00pt"); //added for TD-4006 14.14
-                }
-                else
-                {
-                    _writer.WriteAttributeString("fo:min-height", "14.14pt"); //added for TD-2579
-                }
+	            _writer.WriteAttributeString("fo:min-height", "19.00pt");
```
Comment (gtryus) [inline src/OpenOfficeConvert/OOStyles.cs]: What does setting the min-height do? Should we comment this?

File: src/OpenOfficeConvert/OOStyles.cs [hunk 2]
```diff
@@ -2219,14 +2212,18 @@ private void CreateLeftGuideword(int i)
...
             }
-            _writer.WriteStartElement("text:span");
-            _writer.WriteAttributeString("text:style-name", "MT1");
-            _writer.WriteStartElement("text:variable-get");
-            _writer.WriteAttributeString("text:name", "Left_Guideword_L");
-            _writer.WriteEndElement(); //text:variable-get
-            _writer.WriteEndElement();
 
-            if (_projInfo.ProjectInputType.ToLower() == "dictionary")
+	        if (!(isMirrored && i == 19))
```
Comment (gtryus) [inline src/OpenOfficeConvert/OOStyles.cs]: Is this i == 19 related to setting the minimum height to 19 above? Perhaps we need an explanation of what is happening and why this should only be done for mirrored pages with i == 19.

File: src/OpenOfficeConvert/OOStyles.cs [hunk 3]
```diff
@@ -2310,7 +2307,7 @@ private void SetPageNumberInFrame()
             _writer.WriteAttributeString("fo:min-width", "35pt");
             _writer.WriteAttributeString("draw:z-index", "1");
             _writer.WriteStartElement("draw:text-box");
-            _writer.WriteAttributeString("fo:min-height", "14.14pt");//added for TD-2579
+	        _writer.WriteAttributeString("fo:min-height", "19.00pt");
```
Comment (gtryus) [inline src/OpenOfficeConvert/OOStyles.cs]: Here again we might need some comment to explain why the value of 19 was chosen as a fixed value. Should we get a number from a style sheet?

File: src/OpenOfficeConvert/OOStyles.cs (hunk 1 above)
Comment (vkarthim) [reply src/OpenOfficeConvert/OOStyles.cs]: No, We shouldn't comment this min-height. The default size is needed for frame on Header

File: src/OpenOfficeConvert/OOStyles.cs (hunk 2 above)
Comment (vkarthim) [reply src/OpenOfficeConvert/OOStyles.cs]: The "i == 19" defines the position of the guidewords, "19" position mean to Mirrored page with RightGuideword. In that position, we don't need to insert LeftGuideword.

**Comment by gtryus** [comment]
These changes were improved in another pull request.

**Pull request closed without merging**